sound/soc/sof/mediatek/mt8195/mt8195.h

```diff
@@ -0,0 +1,157 @@
+#define DSP_PDEBUGLS0STAT    0x0018
+#define DSP_PDEBUGLS1STAT    0x001c
+#define DSP_PDEBUGPC         0x0020
+#define DSP_RESET_SW         0x0024
+/*reset sw*/
+#define ADSP_BRESET_SW	 BIT(0)
+#define ADSP_DRESET_SW	 BIT(1)
+#define ADSP_RUNSTALL	 BIT(3)
+#define STATVECTOR_SEL	 BIT(4)
+#define DSP_PFAULTBUS        0x0028
+#define DSP_PFAULTINFO       0x002c
+#define DSP_GPR00            0x0030
+#define DSP_GPR01            0x0034
+#define DSP_GPR02            0x0038
+#define DSP_GPR03            0x003c
+#define DSP_GPR04            0x0040
+#define DSP_GPR05            0x0044
+#define DSP_GPR06            0x0048
+#define DSP_GPR07            0x004c
+#define DSP_GPR08            0x0050
+#define DSP_GPR09            0x0054
+#define DSP_GPR0A            0x0058
+#define DSP_GPR0B            0x005c
+#define DSP_GPR0C            0x0060
+#define DSP_GPR0D            0x0064
+#define DSP_GPR0E            0x0068
+#define DSP_GPR0F            0x006c
+#define DSP_GPR10            0x0070
+#define DSP_GPR11            0x0074
+#define DSP_GPR12            0x0078
+#define DSP_GPR13            0x007c
+#define DSP_GPR14            0x0080
+#define DSP_GPR15            0x0084
+#define DSP_GPR16            0x0088
+#define DSP_GPR17            0x008c
+#define DSP_GPR18            0x0090
+#define DSP_GPR19            0x0094
+#define DSP_GPR1A            0x0098
+#define DSP_GPR1B            0x009c
+#define DSP_GPR1C            0x00a0
+#define DSP_GPR1D            0x00a4
+#define DSP_GPR1E            0x00a8
+#define DSP_GPR1F            0x00ac
+#define DSP_TCM_OFFSET       0x00b0    /* not used */
+#define DSP_DDR_OFFSET       0x00b4    /* not used */
+#define DSP_INTFDSP          0x00d0
+#define DSP_INTFDSP_CLR      0x00d4
+#define DSP_SRAM_PD_SW1      0x00d8
+#define DSP_SRAM_PD_SW2      0x00dc
+#define DSP_OCD              0x00e0
+#define DSP_RG_DSP_IRQ_POL   0x00f0    /* not used */
+#define DSP_DSP_IRQ_EN       0x00f4    /* not used */
+#define DSP_DSP_IRQ_LEVEL    0x00f8    /* not used */
+#define DSP_DSP_IRQ_STATUS   0x00fc    /* not used */
+#define DSP_RG_INT2CIRQ      0x0114
+#define DSP_RG_INT_POL_CTL0  0x0120
+#define DSP_RG_INT_EN_CTL0   0x0130
+#define DSP_RG_INT_LV_CTL0   0x0140
+#define DSP_RG_INT_STATUS0   0x0150
+#define DSP_PDEBUGSTATUS0    0x0200
+#define DSP_PDEBUGSTATUS1    0x0204
+#define DSP_PDEBUGSTATUS2    0x0208
+#define DSP_PDEBUGSTATUS3    0x020c
+#define DSP_PDEBUGSTATUS4    0x0210
+#define DSP_PDEBUGSTATUS5    0x0214
+#define DSP_PDEBUGSTATUS6    0x0218
+#define DSP_PDEBUGSTATUS7    0x021c
+#define DSP_DSP2PSRAM_PRIORITY           0x0220  /* not used */
+#define DSP_AUDIO_DSP2SPM_INT            0x0224
+#define DSP_AUDIO_DSP2SPM_INT_ACK        0x0228
+#define DSP_AUDIO_DSP_DEBUG_SEL          0x022C
+#define DSP_AUDIO_DSP_EMI_BASE_ADDR      0x02E0  /* not used */
+#define DSP_AUDIO_DSP_SHARED_IRAM        0x02E4
+#define DSP_AUDIO_DSP_CKCTRL_P2P_CK_CON  0x02F0
+#define DSP_RG_SEMAPHORE00   0x0300
+#define DSP_RG_SEMAPHORE01   0x0304
```

Comment:
Is it only my browser or we have spaces between the define and value?

I also think if the numbers are aligned at the same tab line, it looks a bit better.